metadata-io/src/main/java/com/linkedin/metadata/search/elasticsearch/query/ESSearchDAO.java

```diff
@@ -317,7 +319,7 @@ public Map<String, Long> aggregateByValue(
       int limit) {
     List<EntitySpec> entitySpecs;
     if (entityNames == null || entityNames.isEmpty()) {
-      entitySpecs = new ArrayList<>(entityRegistry.getEntitySpecs().values());
+      entitySpecs = QueryUtils.getQueryByDefaultEntitySpecs(entityRegistry);
```

Comment:
@RyanHolstien - If there isn't a list of entitySpecs, create a list where they have at least 1 query by default field.